CMakeLists.txt

```
@@ -0,0 +1,137 @@
+cmake_minimum_required(VERSION 3.16.2)
+
+project(i3-ipc++ VERSION 0.8.0 LANGUAGES CXX)
+
+include(CMakePackageConfigHelpers)
+
+find_package(rapidjson REQUIRED)
```

As discussed in the [comments](https://github.com/Iskustvo/i3-ipcpp/pull/7#issuecomment-1065897991), please add this change as well.
```suggestion
find_package(RapidJSON REQUIRED)
```